pkgs/development/haskell-modules/configuration-ghc-8.10.x.nix

```diff
@@ -98,4 +98,10 @@ self: super: {
 
   # Break out of "Cabal < 3.2" constraint.
   stylish-haskell = doJailbreak super.stylish-haskell;
+
+  # Agda 2.6.1.2 only declares a transformers dependency for ghc < 8.10.3.
+  Agda = appendPatch super.Agda (pkgs.fetchpatch {
+    url = "https://github.com/agda/agda/commit/76278c23d447b49f59fac581ca4ac605792aabbc.patch";
+    sha256 = "1g34g8a09j73h89pk4cdmri0nb0qg664hkff45amcr9kyz14a9f3";
+  });
```

Comment:
Could you instead ask an Adga maintainer (or a Hackage maintainer) to make a Hackage revision fixing this?  Doing that would fix this for all Haskellers, instead of just here in nixpkgs.

I'm hoping that a Hackage revision would be able to fix something like this.

Reply:
No, it wouldn’t fix it for other Haskellers, because nixpkgs is already carrying a needed patch for ListLike (a dependency of Agda): see commit 7963b444ca6a3d93ae05ef04cec4d31e55fa44d8.

I’ve asked anyway in a comment on agda/agda#5109, but I would not be surprised if upstream wants to wait for the ListLike fix, ddssff/listlike#8.

Reply:
Thanks for asking upstream.

Feel free to ping me here in a few days if they don't end up getting back to about doing a Hackage revision, and I'll merge this in for you.

Reply:
Upstream: “No. Unfortunately,  `if` conditions can not be changed in Hackage.”

Reply:
Ah, thanks unfortunate.  I was able to confirm that as well with one of my own packages.

Well thanks for going back and forth with me on this.  I'll merge this in.

I imagine this will break the next time a new version for `Agda` is released, so please be on the lookout for that (and feel free to ping me again when you need this patch removed).